src/main/resources/messages.yml

```diff
@@ -58,6 +58,9 @@ en:
   items_taken: '&2Lost {1}&2 x{2}'
   money_given: '&2Received {1} {2}'
   money_taken: '&2Lost {1} {2}'
+  point_given: '&2You receive {1} points to {2}'
+  point_taken: '&2You lose {1} points to {2}'
+  point_multiply: '&2Your points to {2} have been multiplied by {1}'
```

Comment:
Be consistent with tense you use. Some perfect form is probably the better choice over present.

Reply:
If so then it should be `Your points {2} are multiplied by {1}` ?

Reply:
You could also change the point_multiply message, but I especially wanted to point out the two sentences in present tense. `You receive {1} points to {2}` as opposed to `You received {1} points to {2}`. What exact tense to use is flavour, but it should be consistent with the other translations and especially the other translations regarding the same topic.

Reply:
Speaking of tense, the naming here is wrong too. It should be `point_multiplied`

Reply:
i will change the tense to be the same as "items_give"